internal/flow/flow.go

```diff
@@ -350,3 +420,65 @@ func (s *Service) cleanCopy(ctx context.Context, src, dest string) error {
 	}
 	return nil
 }
+
+func zipFiles(files []fileInfo) ([]byte, error) {
+	buf := new(bytes.Buffer)
+	w := zip.NewWriter(buf)
+	for _, file := range files {
+		f, err := w.Create(file.relativePath)
+		if err != nil {
+			return nil, err
+		}
+		fileContent, err := ioutil.ReadFile(file.fullPath)
+		if err != nil {
+			return nil, err
+		}
+		_, err = f.Write(fileContent)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	// Make sure to check the error on Close.
```

Comment:
What's the reasoning for adding this comment? I'm not sure I understand the intent :)

Reply:
Just plain plagiarism 🤦 
https://golang.org/src/archive/zip/example_test.go